modules/alloy/src/test/ts/atomic/position/view/BounderCalcRepositionTest.ts

```diff
@@ -45,8 +46,8 @@ UnitTest.test('BounderCalcRepositionTest', () => {
       const bounds = makeBounds(input.boundsX, input.boundsY, input.boundsW, input.boundsH);
       const output = Bounder.calcReposition(input.newX, input.newY, input.width, input.height, bounds);
 
-      const xIsVisible = output.limitX <= bounds.right() && output.limitX >= bounds.x();
-      const yIsVisible = output.limitY <= bounds.bottom() && output.limitY >= bounds.y();
+      const xIsVisible = output.limitX <= (bounds.right() - input.width) && output.limitX >= bounds.x();
```

Comment:
Probably best to check with Andy because I ported the Bounder code over almost exactly to alloy, but I think initially he was OK with it not being on the screen completely, as long as part of it was on screen. Is that no longer acceptable?

Reply:
This code has already changed a lot since the repartee days.